submissions/reviewer_guide.md

```diff
@@ -1,14 +1,55 @@
 # Guide for Reviewers
 
-Please be respectful and kind to the authors in your reviews. Our [code of conduct](../process/coc) is mandatory for everyone involved in our review process.
+```{epigraph}
+Thank you for taking the time to review a package for pyOpenSci and for contributing
+to making open source scientific software easier to use by the community! If you
+have any questions about the review process, feel free to reach out to one of
+our editors or to post a question on our [discourse forum](https://pyopensci.discourse.group/).  
+```
+
+Please be respectful and kind to the authors in your reviews. Our
+[code of conduct](../process/coc) is mandatory for everyone involved in our
+review process.
+
+## A Guide for New Reviewers
+
+TODO: Add text about the criteria for being a reviewer here ... outline idea below
```

Comment:
if you want to keep this in the text as a TODO, add a `%` before it and it will be a "comment" (AKA, won't make it into the HTML)

Reply:
oh no way!!! that's exactly what i wanted to do. i figure it's easier to merge small changes than shoot for lots of bigs ones but i wanted to flag where help is needed too. thank you!